airbyte-integrations/connectors/source-orb/integration_tests/configured_catalog.json

```
@@ -0,0 +1,259 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "customers",
+        "json_schema": {
```

By the way, all the `json_schema` fields in the integration test catalog can be replaced with an empty object. The full schema will populated automatically.

```suggestion
        "json_schema": {},
```

---

I went ahead and removed the `json_schema` from the integration test catalog as you suggested